sources/packages/web/src/components/common/NoticeOfAssessmentFormView.vue

```diff
@@ -20,6 +20,33 @@
       </p>
     </template></confirm-modal
   >
+  <footer-buttons
+    justify="space-between"
+    primaryLabel=""
+    :showPrimaryButton="true"
+    :showSecondaryButton="false"
+    class="mx-0"
+    v-if="!viewOnly"
+  >
+    <template #primary-buttons>
+      <div><!-- secondary buttons placeholder for the justify to work --></div>
+      <div class="p-0 m-0">
+        <v-btn
+          color="danger"
+          variant="elevated"
+          @click="confirmCancelApplication"
+          >Cancel application</v-btn
+        ><v-btn
+          v-if="assessmentId === currentAssessmentId"
+          class="ml-2"
+          color="primary"
+          @click="confirmAssessment()"
+          :disabled="!canAcceptAssessment"
+          >Accept assessment</v-btn
+        >
+      </div>
+    </template>
+  </footer-buttons>
```

Comment:
The `footer-buttons` can be used out-of-box as show below. It is just missing the color for the secondary button that can follow the same in place for the `primaryButtonColor`.

```vue
 <footer-buttons
    justify="end"
    primaryLabel="Accept assessment"
    :showPrimaryButton="assessmentId === currentAssessmentId"
    :disablePrimaryButton="!canAcceptAssessment"
    @primaryClick="confirmAssessment"
    secondaryLabel="Cancel application"
    :showSecondaryButton="true"
    @secondaryClick="confirmCancelApplication"
    v-if="!viewOnly"
  >
```